dask/dataframe/core.py

```diff
@@ -1452,12 +1452,26 @@ def var(self, axis=None, skipna=True, ddof=1, split_every=False, dtype=None, out
             return handle_out(out, result)
         else:
             num = self._get_numeric_data()
```

Comment:
I know this was here, but I'd prefer we use `self.select_dtypes(include='number')`, if it doesn't break any tests.

Reply:
I think it's going to be `self.select_dtypes(include=['number', np.timedelta64, 'bool'])` to match pandas behavior.

Reply:
Feel free to ignore this if it’s complicated.